Translated/Dialog/Dighal/ssn_012220.csv

```diff
@@ -0,0 +1,11 @@
+ssn_012220_0001#0,"""Been enjoying Gathering?"""
+ssn_012220_0002#0,"""Outside the city, there's a wealth of vegetables, fruits, Ores and Tames meat to be harvested."""
+ssn_012220_0003#0,"""Those fresh ingredients and strong materials are sure to benefit you in your battles to come."""
+ssn_012220_0101#0,"""Yo, how's it going?"""
+ssn_012220_0102#0,"""Now that you've been getting some new weapons and upgrading them a bit, you might notice they'll require some new types of Ore."""
+ssn_012220_0103#0,"""You want to learn more about those Ores?"""
+ssn_012220_0201#0,"""Dualomite is a purple Ore with a jagged formation. It's an Ore similar in colour to Photon Chunks, but with different properties."""
+ssn_012220_0202#0,"""They're both gorgeous gems, but they can also be used at the Item Lab as catalysts for strengthening your equipment."""
+ssn_012220_0203#0,"""Many can be found dotting the walls of the Southern Aerio caverns. It'd do you good to go and try finding some."""
+ssn_012220_0301#0,"""Mm, I see... That's unfortunate."""
+ssn_012220_0302#0,"""Well, whenever you'd like to learn about Ores, let me know at any time."""
```

Comment:
"whenever... at any time" sentence construction is a bit awkward.

Reply:
grammarception fixed